lib/node/spvnode.js

```diff
@@ -73,7 +73,8 @@ class SPVNode extends Node {
       createSocket: this.config.func('create-socket'),
       memory: this.config.bool('memory'),
       selfish: true,
-      listen: false
+      listen: false,
+      agent: this.config.str('agent')
```

Comment:
Minor nit, maybe have this grouped with other configs ? after `memory`.

Reply:
fixed.